packages/sacado/src/Kokkos_DynRankView_Fad.hpp

```diff
@@ -93,7 +98,11 @@ create_mirror(
       Kokkos::LayoutStride >::value >::type * = 0);
 
 template<class Space, class T, class ... P>
+#if KOKKOS_VERSION >= 40499
+typename Impl::MirrorDRViewType<Space,T,P ...>::dest_view_type
+#else
 typename Impl::MirrorDRVType<Space,T,P ...>::view_type
```

Comment:
I know this needed to be updated to `MirrorDRViewType` , but I'm not certain that `dest_view_type` is the correct return type, instead of `view_type`?